src/BizHawk.Client.Common/rewind/ZwinderBuffer.cs

```diff
@@ -26,18 +28,42 @@ public ZwinderBuffer(IRewindSettings settings)
 
 			Size = 1L << (int)Math.Floor(Math.Log(targetSize, 2));
 			_sizeMask = Size - 1;
-			_buffer = new MemoryBlock((ulong)Size);
-			_buffer.Protect(_buffer.Start, _buffer.Size, MemoryBlock.Protection.RW);
+			switch (settings.BackingStore)
+			{
+				case IRewindSettings.BackingStoreType.Memory:
+				{
+					var buffer = new MemoryBlock((ulong)Size);
+					buffer.Protect(buffer.Start, buffer.Size, MemoryBlock.Protection.RW);
+					_disposables.Add(buffer);
+					_backingStore = new MemoryViewStream(true, true, (long)buffer.Start, (long)buffer.Size);
+					_disposables.Add(_backingStore);
+					break;
+				}
+				case IRewindSettings.BackingStoreType.TempFile:
+				{
+					var filename = TempFileManager.GetTempFilename("ZwinderBuffer");
+					var filestream = new FileStream(filename, FileMode.Truncate, FileAccess.ReadWrite);

```

Comment:
I attempted to run this, with a quick modification to force it to use a file, and I got a System.IO.FileNotFoundException exception on ZwinderBuffer line 45.

Reply:
Thanks, I didn't actually test this mode yet.  Will do a bit later today and make whatever corrections are needed.

Reply:
I had used the wrong FileMode; looking at the docs, `Create` is what I want (or maybe `CreateNew`).  The unit tests now pass with `Create` used and the ZWinderBuffer modified to always use TempFile.